server/remote/gitlab/gitlab.go

```diff
@@ -359,12 +359,11 @@ func (g *Gitlab) Status(ctx context.Context, user *model.User, repo *model.Repo,
 	}
 
 	_, _, err = client.Commits.SetCommitStatus(_repo.ID, build.Commit, &gitlab.SetCommitStatusOptions{
-		Ref:         gitlab.String(strings.ReplaceAll(build.Ref, "refs/heads/", "")),
-		State:       getStatus(build.Status),
-		Description: gitlab.String(getDesc(build.Status)),
-		TargetURL:   &link,
-		Name:        nil,
-		Context:     gitlab.String(statusContext),
+		State:       getStatus(proc.State),
```

Comment:
we drop Ref - is this intentional?

Reply:
Yip. Tried both ways and without Gitlab is doing some kind of automatic mapping. By setting this option it would for example not notice that the status should be shown inside a MR.